content/foundations/physical_properties/magnetics_susceptibility_duplicate.rst

```diff
@@ -1,269 +1,216 @@
+space. The permeability of free-space represents the relationship between
+:math:`\vec B` and :math:`\vec H` when the material is non-magnetic. For
+materials in which :math:`\vec M = \kappa \vec H`,  the magnetic permeability
+can be defined in terms of the magnetic susceptibility as follows:
 
 .. math::
-		\vec{M}=\kappa \vec{H}
+    \mu = \mu_0 (1 + \kappa )
 
-Susceptibility :math:`\kappa` is a dimensionless number related to the number
-of individual magnetic dipoles in the medium that can be aligned with the main
-field. Note that as the field increases the number of dipoles that align
-themselves with the field also increases.
 
-Each magnetic particle has it's own magnetic moment. Let :math:`\vec{m_i}` be
-the magnetic moment of the :math:`i'\text{th}` particle. This has an amplitude
-and direction, exactly the same as a small bar magnet.  Consider a volume
-:math:`V`. The total magnetic moment, which is the sum of the individual
-magnetic moments, is
 
- .. math::
-	\vec{m}= \sum_i \vec{m_i}	
+**Relative Permeability**
 
-The magnetization :math:`\vec{M}` :math:`(\text{A/m})` is the dipole moment per
-unit volume. It is given by
+Relative permeability (:math:`\mu_r`) defines the ratio between the magnetic
+permeability of the material and the permeability of free-space:
 
 .. math::
-	\vec{M} = \frac {\vec{m}}{V} = \frac {1}{V} \sum_i \vec{m_i}	
+    \mu_r = \frac{\mu}{\mu_0}
+
+
+According to the above definitions, both magnetic susceptibility and magnetic
+permeability are diagnostic physical properties associated with the magnetic
+characteristics of materials. In the literature, it is common to see physical
+property tables which use :math:`\mu`, :math:`\mu_r`, or :math:`\kappa`. For
+most rocks, the susceptibility is small and charcterizing the magnetic
+properties in terms of :math:`\kappa` is convenient. Parameters used to define
+magnetic properties and their associated units are tabulated below.
+
++--------------------------+----------------+-----------+
+| **Property**             | **Symbol**     | **Units** |
++==========================+================+===========+
+| Magnetic Field Intensity | :math:`\vec H` | A/m       |
++--------------------------+----------------+-----------+
+| Magnetic Flux Density    | :math:`\vec B` | T         |
++--------------------------+----------------+-----------+
+| Magnetization            | :math:`\vec M` | A/m       |
++--------------------------+----------------+-----------+
+| Magnetic Susceptibility  | :math:`\kappa` | *Unitless*|
++--------------------------+----------------+-----------+
+| Magnetic Permeability    | :math:`\mu`    | H/m       |
++--------------------------+----------------+-----------+
+| Relative Permeability    | :math:`\mu_r`  | *Unitless*|
++--------------------------+----------------+-----------+
+
+Some useful conversions for units are:
+
+ .. .. figure:: images/images_duplicates/susceptibility_chart.gif
```

Comment:
this comments out the figure. was that intentional? the sentence does not flow if so.